golos.vesting/parameters.hpp

```diff
@@ -0,0 +1,54 @@
+#pragma once
+#include <common/parameter.hpp>
+#include <eosiolib/singleton.hpp>
+#include <common/config.hpp>
+#include <common/parameter_ops.hpp>
+
+
+namespace golos {
+
+
+using namespace eosio;
+
+struct vesting_withdraw : parameter {
+    uint32_t intervals;
+    uint32_t interval_seconds;
+    uint64_t min_amount;
```

Comment:
please, split `vesting_withdraw` to 2 parameters: move `min_amount` to separate one